.travis-docker.sh

```diff
@@ -36,7 +38,7 @@ echo WORKDIR /home/opam/opam-repository >> Dockerfile
 
 if [ -n "$BASE_REMOTE" ]; then
     echo "RUN git remote set-url origin ${BASE_REMOTE} &&\
-        git fetch origin && git reset --hard origin/master"  >> Dockerfile
+        git fetch origin && git reset --hard origin/$base_remote_branch"  >> Dockerfile
```

Comment:
why do we pull master here and not 2.0.0?

Reply:
e.g. the branch to reset to is different in case we are using opam 1.2.2 or opam 2.0.0.

Reply:
When a BASE_REMOTE is supplied, I don't think you can make any assumptions about the branches in that remote. Hence, I think it should be made explicit.

Reply:
Ha yes I see, I understand the rational.

Reply:
Is there a way to progress this? The default behaviour is to use the master branch of the base remote as before, so I would not see any harm merging this.